README.md

```diff
@@ -1,139 +1,32 @@
-|                          | MEDIUM | Allow Containers with Low and Medium vulnerabilities. |
-|                                           | HIGH  | Allow Containers with Low, Medium & High vulnerabilities. |
-|                                           | ALLOW_ALL | Allow all vulnerabilities.  |
-|                                           | BLOCK_ALL | Block all vulnerabilities except listed in whitelist. |
-|<td rowspan=5>packageVulnerabilityPolicy.maximumFixUnavailableSeverity | LOW | Only allow containers with low unpatchable vulnerabilities. |
-|                          | MEDIUM | Allow Containers with Low and Medium unpatchable vulnerabilities. |
-|                                           | HIGH  | Allow Containers with Low, Medium & High  unpatchaable vulnerabilities. |
-|                                           | ALLOW_ALL | Allow all unpatchable vulnerabilities.  |
-|                                           | BLOCK_ALL | Block all unpatchable vulnerabilities except listed in whitelist. |
-
-### AttestationAuthority CRD
-The webhook will attest valid images once they pass the validity check. This is important because re-deployments can occur from scaling events,rescheduling, termination, etc. Attested images are always admitted in custer.
-This allows users to manually deploy a container with an older image which was validated in past.
-
-To view the attesation authority CRD run,
-```
-kubectl describe crd attestationauthorities.kritis.grafeas.io
-
-# List all attestation authorities.
-kubectl get AttestationAuthority --all-namespaces
-NAMESPACE             NAME             AGE
-qa                    qa-attestator    11h
-```
-
-Here is an example of AttestionAuthority.
 ```yaml
-apiVersion: kritis.github.com/v1beta1
-kind: AttestationAuthority
-metadata:
-    name: qa-attestator
-    namespace: qa
-spec:
-    noteReference: v1alpha1/projects/image-attestor
-    privateKeySecretName: foo
-    publicKeyData: ...
+imageWhitelist:
+- gcr.io/my-project/whitelist-image@sha256:<DIGEST>
+packageVulnerabilityPolicy:
+  maximumSeverity: HIGH
+  whitelistCVEs:
+    providers/goog-vulnz/notes/CVE-2017-1000082
+    providers/goog-vulnz/notes/CVE-2017-1000082
 ```
-Where “image-attestor” is the project for creating AttestationAuthority Notes.
 
-In order to create notes, the service account `gac-ca-admin` must have `containeranalysis.notes.attacher role` on this project.
+## Getting Started
 
-The Kubernetes secret `foo` must have data fields `private` and `public` which contain the gpg private and public key respectively.
+* Learn the concepts in the [Kritis whitepaper](https://github.com/Grafeas/Grafeas/blob/master/case-studies/binary-authorization.md)
+* Get Kritis running with the [Installation guide](install.md)
+* Try the [Tutorial](tutorial.md) to learn how to block vulnerabilities
+* Learn the configuration details in the [Usage guide](usage.md)
```

Comment:
Quick Overview of Kritis Components maybe?

Reply:
I wasn't sure about components vs resources based on what's already documented. How about:

"Read the [Resource Reference](resources.md) to configure and interact with Kritis resources"

Reply:
:+1: